src/Internal.AspNetCore.BuildTools.Tasks/module.props

```diff
@@ -2,8 +2,9 @@
 
   <PropertyGroup>
     <_BuildToolsAssembly>$(MSBuildThisFileDirectory)Internal.AspNetCore.BuildTools.Tasks.dll</_BuildToolsAssembly>
+    <_BuildTasksPrefix></_BuildTasksPrefix>
```

Comment:
Until we get https://github.com/aspnet/BuildTools/issues/297, we have to ensure we don't get assembly conflicts between Internal.AspNetCore.Sdk and KoreBuild. In the former this is set to `<_BuildTasksPrefix>Sdk_</__BuildTasksPrefix>`